coursier.bzl

```diff
@@ -308,16 +308,44 @@ def _generate_coursier_command(repository_ctx):
         # The -noverify option seems to be required after the proguarding step
         # of the main JAR of coursier.
         java = repository_ctx.path(java_home + "/bin/java")
-        cmd = [java, "-noverify", "-jar", coursier]
+        cmd = [java, "-noverify", "-jar"] + _get_java_proxy_args(repository_ctx) + [coursier]
     elif repository_ctx.which("java") != None:
         # Use 'java' from $PATH
-        cmd = [repository_ctx.which("java"), "-noverify", "-jar", coursier]
+        cmd = [repository_ctx.which("java"), "-noverify", "-jar"] + _get_java_proxy_args(repository_ctx) + [coursier]
     else:
         # Try to execute coursier directly
-        cmd = [coursier]
+        cmd = [coursier] + _get_java_proxy_args(repository_ctx)
```

Comment:
Hm, this this work? 

```
→ third_party/coursier/coursier -Dhttp.proxyHost=127.0.0.1
Unrecognized argument: -Dhttp.proxyHost=127.0.0.1
```

Reply:
Good point, I misread https://github.com/coursier/coursier/blob/master/doc/docs/other-proxy.md